### DISCUSSION TITLE
Way for a newcomer to contribute to the project

### DISCUSSION THREAD

**Discussion by nickolasoverlay** [pulsar-edit/.github#105] [Pulsar Support]
Hi y'all!
I'm glad that somebody decided to continue Atom's legacy in a form of new, modern and hackable text-editor. Atom was the first editor i've ever used, so i have a special relationship with it.

Please, let me know if there is any bug / feature i could work, to help the project. As an engineer, i can help you with something related to the actual code, i'm not that good at design.

**Comment by kaosine** [comment]
Hmm I think we should have something in the org Readme but don't quote me on that.... We really ought to start labeling stuff with "good first issue" or something tbh 🤔

**Comment by confused-Techie** [comment]
I'll try and give a better answer when I'm back at my computer.

But yes, please feel free to look at our Org readme. We include several links to open issues or feature requests you can take a look at. Or links to our project boards that also have features waiting to be implemented.

Otherwise, to better direct you, what languages are you familiar with? What part of the project are you passionate about? (As in main editor, packages, backend, docs)

**Comment by diypa571** [comment]
Pulsar, it is good and it works, using Linux Iam just happy we have this software to use

**Comment by Flynn81** [comment]
This page linked to from the org's main page might be a good place to start (just starting to look at contributing as well) https://github.com/pulsar-edit/.github/blob/main/project-birth/CONTRIBUTING-DURING-START.md
